WORKSPACE

```diff
@@ -16,9 +16,45 @@ workspace(name = "io_opentelemetry_cpp")
 
 # Load our direct dependencies.
 load("//bazel:repository.bzl", "opentelemetry_cpp_deps")
+load("//bazel:openssl_repositories.bzl", "openssl_repositories")
 
 opentelemetry_cpp_deps()
 
+openssl_repositories()
+
+# Load prometheus C++ dependencies.
+load("@com_github_jupp0r_prometheus_cpp//bazel:repositories.bzl", "prometheus_cpp_repositories")
+
+prometheus_cpp_repositories()
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
I'm not sure, but is `bazel_skylib` the internal module of bazel? Is it necessary to load it from github?

Reply:
It's needed [here](https://github.com/open-telemetry/opentelemetry-cpp/blob/610193c563e7c85a98f11dbbdf588f2f583a372d/bazel/openssl_repositories.bzl#L9).

Reply:
Thanks for explain